core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-device-selector-seo/dot-device-selector-seo.component.stories.ts

```diff
@@ -15,7 +15,7 @@ import { PanelModule } from 'primeng/panel';
 
 import { DotDevicesService, DotMessageService } from '@dotcms/data-access';
 import { CoreWebService, CoreWebServiceMock } from '@dotcms/dotcms-js';
-import { DotIconModule, DotMessagePipeModule } from '@dotcms/ui';
+import { DotIconModule, DotMessagePipe } from '@dotcms/ui';
```

Comment:
Not part of this issue, but seemed like it was needed

Reply:
Oh I also nedeed this. I added it in my [PR](https://github.com/dotCMS/core/pull/25511) too